includes/class-wc-siftscience-admin.php

```diff
@@ -152,36 +155,36 @@ private function output_settings_debug() {
 		}
 
 		private function output_settings_reporting() {
-			if ( isset( $_GET[ 'reset_guid' ] ) ) {
-				if ( $_GET[ 'reset_guid' ] == 1 ) {
+			if ( isset( $_GET['reset_guid'] ) ) {
```

Comment:
Let's go ahead and simplify this to:

```
if ( isset( $_GET['reset_guid'] &&  '1' === $_GET['reset_guid']  ) )
```

Reply:
k

Reply:
actually this well cause an error because if it was not set it cannot evaluate the second part of the expression

Reply:
No it won't. If the first part of the if-statement is `false`, then the second half will not be executed.

example:

```
function test($i) { echo $i; return false; }

$result = (test(1) && test(2) );
```

The output will be:

```
1
```

Reply:
Similarly:

```
function test($i) { echo $i; return true; }

$result = (test(1) || test(2) );
```

Output:

```
1
```

Reply:
your responsibility